### PR TITLE
feat(ER-next): welcome page for post purchase magic link callback

### PR DESCRIPTION
**Pull request by DrShpongle**
<details>
  <summary>Individual</summary>
  
</details>
<details>
  <summary>Individual (dark)</summary>
  
</details>
<details>
  <summary>Team</summary>
  
</details>
<details>
  <summary>Team (dark)</summary>
  
</details>

Episode 4 Nbc GIF by Brooklyn Nine-Nine

### CODE REVIEW

File: apps/epic-react/src/config.ts [hunk 1]
```diff
@@ -66,8 +66,7 @@ const config: DefaultSeoProps & {author: string} = {
     },
     images: [
       {
-        url: `${process.env.NEXT_PUBLIC_URL}/api/og/default`,
-        // url: `${process.env.NEXT_PUBLIC_URL}/og-images/og-image@2x.jpg`,
+        url: `${process.env.NEXT_PUBLIC_URL}/og-images/og-image@2x.jpg`,
```
Comment (DrShpongle) [inline apps/epic-react/src/config.ts:69]: fixed og image url (was missed in previous PR)

File: apps/epic-react/src/components/fancy-button.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      {buttonContent}
+    </MotionLink>
+  ) : (
+    <motion.button type={type} {...mandatoryProps} {...rest}>
+      {buttonContent}
+    </motion.button>
+  )
+}
+
+export default FancyButton
```
Comment (DrShpongle) [inline apps/epic-react/src/components/fancy-button.tsx:68]: Refactored and moved to the standalone component the fancy button Zac created and used in subscribe CTA

File: apps/epic-react/src/pages/welcome.tsx [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+        className="font-heading flex items-center gap-2 self-start rounded-full bg-black px-5 py-2.5 font-semibold text-white transition hover:bg-[#333333] dark:bg-white dark:text-black dark:hover:bg-[#efefef]"
+      >
+        <XIcon /> Share with your friends!
+      </a>
+    </div>
+  )
+}
+
+export const XIcon = () => (
+  <svg
```
Comment (DrShpongle) [inline apps/epic-react/src/pages/welcome.tsx:307]: X icon instead of outdated bird icon for Twitter

**Review by jbranchaud — APPROVED** [review]
looks great!

**Pull request merged**